src/XeroPHP/Enums/PayrollAU/LeaveApplications/LeavePeriodStatusCode.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace XeroPHP\Enums\PayrollAU\LeaveApplications;
+
+class LeavePeriodStatusCode
+{
+    const SCHEDULED = 'Scheduled'; // The default status
```

Comment:
these should be uppercase? - https://developer.xero.com/documentation/api/payrollau/types-and-codes#leaveapplications